### ISSUE TITLE
Incompatible with html-webpack-plugin v4

### ISSUE THREAD

**Issue by dominykas** [GoogleChromeLabs/critters#52]
When used with latest version of `html-webpack-plugin`, it crashes with `Error: Could not find HTML asset.`

It seems the `html-webpack-plugin` no longer uses the file name as the key in the `assets` object?

I can also see there's a #43 - anything that can be done to move this forward?

**Comment by dominykas** [comment]
Just spotted that this may just be a duplicate of #5?

**Comment by vitaliyhayda** [comment]
+1 same issue

**Comment by mrlss** [comment]
+1, would be great to fix this issue

**Comment by blackbird91** [comment]
+1

**Comment by developit** [comment]
@dominykas - if you're able to test that PR against your codebase that would be useful. I will try to find some time as well.

**Comment by dominykas** [comment]
Thanks, will do. Wasn't sure if that PR is usable.

**Comment by dominykas** [comment]
I can't get that to build: `Error: Plugin 'jsx' not found`... Deps are somewhat out of date, so maybe it is that? I'll try to update the deps and see if that makes it compile, unless someone else gets there first.

**Comment by dominykas** [comment]
I patched the `dist` file manually and tested on one of the repos - the HTML output is identical as before (for practical purposes), so it seems to me that it works 👍.

**Comment by dominykas** [comment]
Upgrading `microbundle` makes the `jsx` error go away, but produces a 7MB file in `dist`, which is probably not what needs to happen here, so I think I'll leave this be, unless there's something specific that you want me to do.

**Comment by developit** [comment]
@dominykas yep, I can take care of that. It needs the new `microbundle --target node` flag.

**Comment by mrlss** [comment]
@developit is there any changes this issue will be fixed?

**Comment by buddalasunil999** [comment]
+1

**Comment by DUBANGARCIA** [comment]
+1

**Comment by DUBANGARCIA** [comment]
+1

**Comment by stnwk** [comment]
Would love to see it fixed too! Currently makes it impossible to use with nextjs :/

**Comment by blephy** [comment]
+1

**Comment by weihongyu12** [comment]
+1

**Comment by rastalamm** [comment]
+1

**Comment by DUBANGARCIA** [comment]
+1

**Comment by BzenkoSergey** [comment]
+1

**Comment by blephy** [comment]
+10000 !!!!!

**Comment by lili21** [comment]
https://github.com/GoogleChromeLabs/critters/pull/43
